src/main/java/uk/gov/companieshouse/service/AppealService.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.companieshouse.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.companieshouse.exception.AppealNotFoundException;
+import uk.gov.companieshouse.model.Appeal;
+import uk.gov.companieshouse.model.CreatedBy;
+import uk.gov.companieshouse.repository.AppealRepository;
+
+import java.time.LocalDateTime;
+import java.util.Optional;
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class AppealService {
+
+    private final AppealRepository appealRepository;
+
+    public String saveAppeal(String companyId, Appeal appeal, String ericIdentity) throws Exception {
+
+        appeal.setCreatedBy(CreatedBy.builder().id(ericIdentity).build());
```

Comment:
have we verified that the `ericIdentity` field is the same value as the userId we get back from the account service? i.e. the `id` field inside the `user_profile` object in session? Cause the user's ID is what we want to store here in the createBy field

Reply:
discussed and received clarification the `eric-identity` is the `userId`, but will add extra logging to confirm this